src/main/java/com/actiontech/dble/services/manager/handler/UpdateHandler.java

```diff
@@ -189,12 +194,12 @@ public ManagerWritableTable getWritableTable(MySqlUpdateStatement update, Manage
         return (ManagerWritableTable) managerBaseTable;
     }
 
-    private int updateRows(ManagerService service, ManagerWritableTable managerTable, Set<LinkedHashMap<String, String>> affectPks, LinkedHashMap<String, String> values, PacketResult packetResult) throws Exception {
```

Comment:
why remove packet result?

Reply:
use ReloadException.java, uniform capture at the outermost layer method